xbmc/utils/HevcSei.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "BitstreamReader.h"
+
+#include <optional>
+#include <vector>
+
+class CHevcSei
+{
+public:
+  CHevcSei() = default;
+  ~CHevcSei() = default;
+
+  uint8_t payload_type;
```

Comment:
```suggestion
  uint8_t payload_type{0};
```

This and others. 
Initializing inside the class seems safer in order to avoid "missing initialization" mistakes when class is used.